packages/shared/src/hooks/usePersonaContext.ts

```diff
@@ -25,8 +25,10 @@ function usePersonaInformation(
     queryOwnedPersonaInformation?: (initializedOnly: boolean) => Promise<PersonaInformation[]>,
 ) {
     const { data: personas = EMPTY_LIST, refetch } = useQuery({
-        queryKey: ['my-own-persona-info'],
+        queryKey: ['@@my-own-persona-info'],
```

Comment:
You should also change the following files:

- packages/mask/content-script/components/InjectedComponents/SetupGuide/VerifyNextID.tsx
- packages/mask/popups/pages/Personas/AccountDetail/index.tsx (line 86 and 157)

@swkatmask